CONTRIBUTING.md

```diff
@@ -179,7 +179,36 @@ virtualenv venv
 # Install openhtf into the virtualenv in dev mode.
 python setup.py develop
 ```
+### MacOS
```

Comment:
I'd put an empty line here to make the .md more readable, even if it actually _renders_ the same either way.